src/Experimental/ShortestPaths/astar.jl

```diff
@@ -0,0 +1,126 @@
+# Parts of this code were taken / derived from Graphs.jl. See LICENSE for
+# licensing details.
```

Comment:
Maybe it would be time to get rid of code that is taken from Graphs.jl and do some "cleanroom" implementation.

Reply:
Can't. The algorithms themselves were derived from the Graphs.jl implementation. There is also no harm in crediting them. The package is deprecated/effectively abandoned and there are no licensing issues.